### DISCUSSION TITLE
Print or Export to PDF with CoverSheet

### DISCUSSION THREAD

**Discussion by mobab-th** [sepinf-inc/IPED#588] [Ideas]
Useful would be a possibility to print a file (document, e-mail, ..) with a cover sheet with metadata to the file or export it as PDF.

**Comment by mobab-th** [comment]
hi,

I have to come back to my suggestion.
Is there a way to print all metadata of a file at once? If not, such an option would be very helpful.

**Comment by lfcnassif** [comment]
Hi @mobab-th, AFAIK there is not. I created #1039, but it has no priority currently and, as I understood, it is different then what you are asking for...

**Comment by lfcnassif** [comment]
Exporting all metadata of a file to CSV would help? There is a PR about this waiting for review...

**Comment by mobab-th** [comment]
A printout / export in the table representation as in the metadata tab is more what I have in mind.
The usual procedure of my colleagues is to export a relevant document and to export / print the metadata of this document at the same time. A table with the metadata would be quite helpful.
